node_modules/oae-util/lib/oae.js

```diff
@@ -61,6 +61,7 @@ module.exports.init = function(config, callback) {
 
     // Set up application-level error handler
     process.on('uncaughtException', function(err) {
+        log().error(err.stack);
```

Comment:
These logging statements could be swapped around so that the stack trace shows up below the message.